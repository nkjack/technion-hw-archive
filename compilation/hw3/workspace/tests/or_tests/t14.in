int a(int c){
return 3;
}

void main(){
int t1 = 1;
int t2 = 10;
int c = a(t1+t2);
}
