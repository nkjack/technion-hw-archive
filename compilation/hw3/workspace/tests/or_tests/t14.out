---end scope---
a - 0 preconditions
c INT -1
---end scope---
main - 0 preconditions
t1 INT 0
t2 INT 1
c INT 2
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
a (INT)->INT 0
main ()->VOID 0
